dev/reformat

```diff
@@ -3,18 +3,18 @@
 set -x
 
 # python style checks rely on `ruff` in path
-if ! command -v ruff &> /dev/null; then
-    echo "Installing Ruff ..."
-    pip install ruff
+if ! command -v ruff &>/dev/null; then
+	echo "Installing Ruff ..."
+	pip install ruff
```

Comment:
Unnecessary and unrelated changes, which introduced incorrect indent. Please leave it untouched.

Reply:
Updated.